src/components/summary/SummaryGroupComponent.tsx

```diff
@@ -94,27 +84,15 @@ function SummaryGroupComponent({
 }: ISummaryGroupComponent) {
   const classes = useStyles();
 
-  const [title, setTitle] = React.useState<string>('');
-  const [groupHasErrors, setGroupHasErrors] = React.useState<boolean>(false);
-  const [groupChildComponents, setGroupChildComponents] = React.useState<string[]>([]);
-
-  const groupComponent = useAppSelector(
-    (state) =>
-      (state.formLayout.layouts &&
-        pageRef &&
-        componentRef &&
-        getComponentForSummaryGroup(state.formLayout.layouts[pageRef], componentRef)) ||
-      undefined,
-    shallowEqual,
-  );
+  const node = useResolvedNode(componentRef);
+  const textResourceBindings = node?.item.textResourceBindings;
+
+  const removeExcludedChildren = (n: AnyNode<'resolved'>) =>
+    !excludedChildren ||
+    !excludedChildren.includes(n.item.id) ||
```

Comment:
Does `n.item.id` include multiPageIndex?

Reply:
Is it not correct that it should not contain it?

Reply:
Whoops, I misunderstood completely.. It's true that there was no backwards compatibility issue here. The `componentDefinition.id` property has never included the `multiPageIndex`, but items in `children` _might_. I.e., if you pass `excludedChildren={someGroup.children}`, you'd expect every child to be excluded, and they would - except if `someGroup.edit.multiPage === true`.

So I was thinking I might add a check for `excludedChildren.includes('${n.item.multiPageIndex}:${n.item.id}')` as well - to make multiPage groups `children` property work the same as non-multiPage groups. The more I think about it, I don't know if it's a valid/needed use case, so we should probably just skip it.

Reply:
Its probably not needed